app/views/layouts/_marketplace_head.haml

```diff
@@ -99,3 +99,7 @@
 
 - unless on_admin?
   = @current_community.custom_head_script.try(:html_safe)
+
+- if on_admin? && IntercomHelper.admin_intercom_respond_enabled?
+  - user_uuid = @current_user.uuid_object.to_s
+  = render partial: "layouts/intercom", locals: { app_id: IntercomHelper.admin_intercom_app_id, email: @current_user.primary_email&.address, user_id: @current_user.id, user_uuid: user_uuid, name: PersonViewUtils.person_display_name(@current_user, @current_community), marketplace_id: @current_community.id, marketplace_uuid: @current_community.uuid_object.to_s, marketplace_url: @current_community.full_url, user_hash: IntercomHelper.user_hash(user_uuid) }
```

Comment:
The reason is that this piece of code is part of the main layout. In normal case I think setting the locals in controller and passing them to the view is the perfect way to go, but since there's no one single controller, it's a bit trickier. One thing that could be done is to set the variables in controller instance variables in ApplicationController's before filter, but I'm not sure that's any better, since the application controller is already quite big and the list of filters is long.